### DISCUSSION TITLE
Center screen size

### DISCUSSION THREAD

**Discussion by Depechie** [rxhanson/Rectangle#1531] [General]
Is there a way to change the default window size for the Center option?
I love the center option, but I always have to do 2 or 3 Larger options after that to make it ok for my ultra wide screen... would be great if we could alter the default % that it currently uses for it's margins around the window.

**Comment by rxhanson** [comment]
To achieve this you can use Terminal commands to either adjust the `Almost Maximize` size or `Add an extra centering command with custom size`:

https://github.com/rxhanson/Rectangle/blob/main/TerminalCommands.md#almost-maximize

Restart the app after executing Terminal commands.
